e2e/cypress/integration/messaging/ctrl_cmd_k_find_gm_by_matching_name_spec.js

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***********************************************************  ****
+
+// Group: @messaging
+
+describe('Messaging', () => {
+    let testTeam;
+    let firstUser;
+    let secondUser;
+    let thirdUser;
+
+    before(() => {
+        // # Login as test user
+        cy.apiInitSetup().then(({team, user}) => {
+            firstUser = user;
+            testTeam = team;
+
+            // # Create two more users
+            cy.apiCreateUser().then(({user: user1}) => {
+                secondUser = user1;
+                cy.apiAddUserToTeam(testTeam.id, secondUser.id);
+            });
+
+            cy.apiCreateUser().then(({user: user2}) => {
+                thirdUser = user2;
+                cy.apiAddUserToTeam(testTeam.id, thirdUser.id);
+            });
+
+            cy.apiLogin(firstUser);
+        });
+    });
+
+    it('MM-T1226 - CTRL/CMD+K - Find GM by matching username, full name, or nickname, even if that name isn\'t displayed', () => {
+        // # Create a group channel and add the three users created in the 'before' hook
+        cy.apiCreateGroupChannel([firstUser.id, secondUser.id, thirdUser.id]).then((groupChannel) => {
+            // # Visit the newly created group message
+            cy.visit(`/${testTeam.name}/channels/${groupChannel.body.name}`);
+
+            // # Go to town square
+            cy.get('#sidebarItem_town-square').click();
+
+            // # Type either cmd+K / ctrl+K depending on OS and type in the first 7 of the second user's last name
+            cy.get('#post_textbox').cmdOrCtrlShortcut('K');
+            cy.focused().type(`${secondUser.last_name.slice(0, 7)}`);
+
+            // * The suggestion for the GM channel with the user that was searched for should show the username of the user
+            cy.get('.suggestion--selected').should('exist').and('contain.text', secondUser.username);
+
+            // * The suggestion for the GM channel with the user that was searched for should show the 'G' text
+            cy.get('#suggestionList').find('div').eq(1).findByText('G').should('exist');
```

Comment:
I'm good as is but would be nice to combine this to above so that it's clear that it refers to the same selected suggestion, like:
```javascript
cy.get('.suggestion--selected').should('exist').
    and('contain.text', secondUser.username).
    findByText('G').should('exist');
```

Reply:
This is much more concise. Thanks pare!